build.zig

```diff
@@ -158,6 +158,12 @@ pub fn build(b: *std.Build) !void {
         "Build a Position Independent Executable. Default true for system packages.",
     ) orelse system_package;
 
+    const strip = b.option(
+        bool,
+        "strip",
+        "Strip the final executable. Default true for fast and small releases",
+    ) orelse null;
```

Comment:
`orelse null` doesn’t make a lot of sense, since `option` already returns an optional.

i think the best solution is to move that switch into the orelse here.

Reply:
good point, addressed in 061a730dd37188c6b6aca7eac66f8d0a9bc108b3